.parcelrc

```diff
@@ -23,6 +23,7 @@
     "*.html": "parcel-packager-ssg"
   },
   "optimizers": {
-    "*.js": ["@parcel/optimizer-terser"]
+    "*.js": ["@parcel/optimizer-terser"],
+    "**/spectrum-theme.cjs": ["parcel-optimizer-strict-mode"]
```

Comment:
The exact path of `packages/@react-spectrum/s2/style/dist/spectrum-theme.cjs`doesn't work, but this works.